github-issue-opener/iac/main.tf

```
@@ -9,6 +9,10 @@ terraform {
   }
 }
 
+variable "ko_build_importpath" {
+  default = "github.com/chainguard-dev/enforce-events/github-issue-opener/cmd/app"
+}
```

```suggestion
locals {
  importpath = "github.com/chainguard-dev/enforce-events/github-issue-opener/cmd/app"
}
```

Then below `${local.importpath}`

---

🤦  when I tried that with `"gcr.io/${var.project_id}/${locals.importpath}"` and got an error and I reached for `variable`.. because local, locals, tomato potato.